docs/developer-guide/project-conventions.md

```diff
@@ -79,14 +77,34 @@ We follow a loose version of the [Git flow branching model](https://nvie.com/pos
 - Release branches should always be based on: `develop`
 - Release branches should be merged via PR into `main`, followed by PR to merge `main` into `develop`.
 
+**See below for merging guidance**
+
 ### Naming branches
 
-- Branch names for ticketed new features should start with: `feature/<JIRA-TICKET-with-short-description>`.
-- Branch names for ticketed bugs should start with: `fix/<JIRA-TICKET-with-short-description>`.
-- Branch names for release should start with: `release/<major.minor.patch>`.
-- Branch names for housekeeping tasks or other unticketed work should start with: `chore/<short-description>`
-- Ticket numbers should be included in branch names wherever possible. For example:
-  - `feature/DF-123-extra-squiggles`
-  - `fix/DF-999-image-view-error`
-  - `chore/short-description`
-- Stick with alphanumeric characters and hyphens where possible and avoid random special characters.
+- Use only alphanumeric characters and hyphens where possible and avoid special characters.
+- Branch names for ticketed new features should follow: `feature/JIRA-TICKET-NUMBER-with-short-description`
+- Branch names for ticketed bug fixes should follow: `fix/JIRA-TICKET-NUMBER-with-short-description`
+- Branch names for releases should follow: `release/major.minor.patch`
+- Branch names for housekeeping tasks or other unticketed work should follow: `chore/short-description`
+- For example:
+    - `feature/UN-123-extra-squiggles`
+    - `fix/DF-999-image-view-error`
+    - `release/1.0.0`
+    - `chore/update-documentation`
+
+### Naming pull requests
+
+- Pull requests for features and bug fixes should be titled: `JIRA-TICKET-NUMBER: short-description`
+- Pull requests for release branches should be titled: `Release X.X.X into main`
+- Pull requests for housekeeping tasks or other unticketed work should be titled: `CHORE: short-description`
+- For example:
+    - `UN-123: Add extra squiggles`
+    - `DF-999: Fix image view error`
+    - `Release 1.0.0 into main`
+    - `CHORE: Update documentation`
+
+### Merging branches
+
+- When merging a feature branch into `develop`, use the `Squash and merge` option to keep the commit history clean.
+- When merging a release branch into `main`, use the `Merge commit` option to keep the commit history continuous from `develop`.
+- When merging `main` back into `develop` (after merging a release branch into `main`), use the `Merge commit` option to prevent any conflicts when merging future releases into `main` to keep the history in sync.
```

Comment:
Add one at the beginning of these points
- Ensure the branch has the latest changes from the base branch as it may be out of sync between PR submission/rework/reviews. Check with the developer if it is not.

_If the reviewer clicks on `Update branch`, then the reviewer becomes a contributor and there may be a situation where everything looks fine - but a possible mix-up for migrations. Therefore I thought to _add this note here_  as also the timing between code-reword-review will makes the branch out of sync with the base branch. Thoughts?_